clouditor-engine-core/src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version = "1.0" encoding = "utf-8"?>
```

Comment:
Is the XML necessary if its any way empty? :D

Reply:
I didn´t found a way to get rid of it...